src/web/styles.scss

```diff
@@ -267,3 +267,13 @@ h6,
   font-weight: bold;
   color: blue;
 }
+
+.handsontable .invalid-row {
```

Comment:
Just wondering if red background with black text is actually good UX as the instructor would like to spot the error in text more obviously?
I would propose something like a contrasting colour e.g. white, but subject to discussion.

Reply:
I used red background and white text for highlighting errors